### PR TITLE
#109 - Add stylelint-webpack plugin with base stylelint configuration

### PR DESCRIPTION
**Pull request by robwozniak**

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -34,5 +36,13 @@ module.exports = (storybookBaseConfig, configType, defaultConfig) => {
     }
   })
 
+  defaultConfig.plugins.push(new StyleLintPlugin({
+    files: [
+      'components/**/*.vue',
+      'assets/styles/**/*.scss'
+    ],
```
Comment (szafran89) [inline .storybook/webpack.config.js]: we have also `scss` styles in components directory. e.g `/components/02-elements/button/Button.scss` We are going to divide all components to separate files so I'm not sure if you cover these files?

**Comment by szafran89** [comment]
Now we have failing checks. Please fix it ;)

**Comment by robwozniak** [comment]
Ok, I thought that we can handle that in the other task :smile:

File: .storybook/webpack.config.js (hunk 1 above)
Comment (robwozniak) [reply .storybook/webpack.config.js]: Good catch! These files were omitted :+1:

**Pull request merged**